AC3.2-MidUnit-6-Exam/DoubleVideoViewController.swift

```diff
@@ -7,25 +7,97 @@
 //
 
 import UIKit
+import AVKit
+import AVFoundation
+import MobileCoreServices
 
-class DoubleVideoViewController: UIViewController, CellTitled {
-    var titleForCell: String = "Double Video"
 
+class DoubleVideoViewController: UIViewController, CellTitled, UIImagePickerControllerDelegate, UINavigationControllerDelegate {
+    var titleForCell: String = "Double Video"
+    var movieURL: URL?
+    
+   
+    
     @IBOutlet weak var videoContainerTop: UIView!
     @IBOutlet weak var videoContainerBottom: UIView!
     
+    var player: AVPlayer?
+    
+    
     override func viewDidLoad() {
         super.viewDidLoad()
 
         // don't f with this line
         self.edgesForExtendedLayout = UIRectEdge(rawValue: 0)
     }
-
+    
     override func viewDidLayoutSubviews() {
         // update your layers' frames here
+        guard let sublayerTop = self.videoContainerTop.layer.sublayers,
+            let sublayerBottom = self.videoContainerBottom.layer.sublayers
+            else {
+                return
+        }
+        
+        for layerTop in sublayerTop {
+            layerTop.frame = self.videoContainerTop.bounds
+        }
+        
+        for layerBottom in sublayerBottom {
+            layerBottom.frame = self.videoContainerBottom.bounds
+        }
     }
-    
+
     @IBAction func loadVideo(_ sender: UIButton) {
+        let imagePickerController = UIImagePickerController()
+        imagePickerController.sourceType = .photoLibrary
+        imagePickerController.mediaTypes = [String(kUTTypeMovie)]
+        imagePickerController.delegate = self
+        let view = UIImagePickerController()
```

Comment:
There was a warning about this unused variable. Also, the name ```view``` has some issues:

* ```UIViewController``` has a property named ```view``` that will be masked by this local constant for the rest of that function, which is probably not what you'd expect.
* It also doesn't reflect the type it refers to (a view controller).

This actually didn't break any functionality but it more a matter of good code quality and style.